### DISCUSSION TITLE
Form fields in widget don't cast to boolean

### DISCUSSION THREAD

**Discussion by notflip** [statamic/cms#7049] [Troubleshooting] [answered]
I'm using a radio button in my form, to capture someone attending/not attending. As values I entered true/false for this.

![Screenshot 2022-11-14 at 15 37 30](https://user-images.githubusercontent.com/4146384/201687458-144ab2ba-fb6f-46da-959a-0d6b0ddc9a06.png)

In the dashboard I'm using a widget (set in cp.php), with the following data
```
    'widgets' => [
        [
            'type' => 'form',
            'form' => 'rsvp',
            'fields' => ['names', 'email', 'attending', 'message'],
            'limit' => 10,
        ],
    ],
```

As you can see in the screenshot this is not being casted.

![Screenshot 2022-11-14 at 15 38 14](https://user-images.githubusercontent.com/4146384/201687645-22842ebc-5c15-4528-8fea-d34e08fae04a.png)

- Cast booleans is enabled for the radio button in the blueprint

**Comment by edalzell** [comment, marked as answer]
maybe try `'true'` and `'false'`?

**Reply by notflip** [reply]
That's it! Thanks
